l10n/en/firefox/more/misinformation.ftl

```diff
@@ -0,0 +1,64 @@
+misinformation-overly-sensational = Is the headline overly sensational?
+misinformation-check-out-websites = Check out websites and news outlets that you trust to see if and how they report on the same topic. Content that is intended to manipulate will often use extremes and exaggerations for which there is no evidence and will not be found in well-researched reporting.
+misinformation-content-up-to-date = Is the content up to date?
+misinformation-outdated-information = Outdated information is not necessarily meant to disinform, but it may not be factually correct any longer.
+misinformation-credible-publication = Did the author publish other articles or posts, ideally also in credible publications?
+misinformation-authors-name = If you only find the author's name on questionable sites, you should not pay too much importance to their statements.
+misinformation-images-captions = Do images and captions match?
+misinformation-images-are-powerful = Images are a very powerful tool for targeted disinformation. Depending on the caption, the perception can be twisted into the absolute opposite. If you are unsure, perform a reverse image search to find out how the image is described on other sites.
+misinformation-how-firefox-helps-heading = How { -brand-name-firefox } helps you avoid misinformation
+misinformation-firefox-keeps-trackers-heading = { -brand-name-firefox } keeps trackers in check:
+
+# Variables:
+#   $firefox (url) - link to https://www.mozilla.org/firefox/new/
+#   $protection (url) - link to https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop
+misinformation-firefox-keeps-trackers-desc = As mentioned before, tracking plays a big role in the spread of misinformation. <a href="{ $firefox }">Using { -brand-name-firefox }</a> with <a href="{ $protection }">Enhanced Tracking Protection</a> helps by blocking third-party trackers from many advertisers and other players by default when you browse the web, so that you gain back control over your online data.
+
+misiinformation-social-media-clean-heading = { -brand-name-firefox } helps to keep your social media feeds clean:
+
+# Variables:
+#   $container (url) - link to https://www.mozilla.org/firefox/facebookcontainer/
+misiinformation-social-media-clean-desc = Social networks know a lot about you. Plus, they’re able to track you beyond their platforms with their Share and Like buttons despite Tracking Protection — even if you don’t have an account. The <a href="{ $container }">{ -brand-name-facebook-container }</a> for { -brand-name-firefox } makes sure that { -brand-name-facebook } and { -brand-name-instagram } can't do this so easily, which again reduces the probability of seeing misinformation via ads and promoted content significantly.
+
+misinformation-surface-content-heading = { -brand-name-firefox } helps you surface content that’s worthy of your time:
```

Comment:
Since this is only shown for English and German, should we omit it from Fluent and hardcode it in the template? Or if we want to have translations handled in Pontoon maybe just add a comment that it's only shown in EN and DE?

Reply:
This is shown the en and de right now since only those locales get Pocket Recommendations. I don't have the German translation yet, so can't hard code it alas. But I guess there's also a possibility that this might include fr at some point in the future. 

I'll add a comment to the string 👍